utils/webdriverConfigs/chromeDriver.js

```diff
@@ -0,0 +1,22 @@
+const { Builder } = require('selenium-webdriver');
+
+const driver = new Builder().forBrowser('chrome').build();
+
+async function chromeConfig() {
+  await driver.manage().window().maximize();
+}
+
+async function quit() {
+  return driver.quit();
+}
+
+async function sleep() {
```

Comment:
What is the difference between this function and the` wait()` method of your `BaseElement` class?

Reply:
First: wait rely on an element
Second: sleep is used for service debugging and development.